### ISSUE TITLE
Rewrite introduction to docs, removing or explaining jargon terms

### ISSUE THREAD

**Issue by andrewphilipsmith** [Living-with-machines/MapReader#117]
- [x] Define the meaning of ‘pipeline’ in layman’s terms
- [ ] Examples with pics of what MapReader can produce as outputs
- [x] Mapreader pipeline subsection - summarise in a sentence or two what each of the following two steps means. It should be very simple and immediately understandable to something without data science knowledge.
    - [x] preprocessing/annotation
    - [x] training/inference
- [x] In the image, state what ‘DB’ is below on in the image. I’m not sure that a lot of novices will know that it means database. You can also interchange it with ‘data repositories’ or something more intuitive.
- [x] In the ‘What is included’ subsection, the first mention of the word ‘patch’ should be explained. Infact, I’d review and rewrite this section to provide more clarity for first time users.

**Comment by kallewesterling** [comment]
Fixed in documentation. Final review by @kmcdono2

**Comment by rwood-97** [comment]
> What are outputs of mapreader? patches? model? classified images?
- [ ] Add pic for input and output for every step in the pipeline
